NYPL/fedora_ingest_rails
# DR 2597 deploy to both qa environments

## Jira Tickets ##
[Fedora Ingest Migration: Pushes to qa branch deploy to migrated and unmigrated environments](https://jira.nypl.org/browse/DR-2597)

## Things Done ##
- Updated the travis script and provisioning scripts to deploy into nypl-dams-dev from the qa branch

## How to Test ##
- See the successful qa deploy at https://app.travis-ci.com/github/NYPL/fedora_ingest_rails/builds/266799232
- Go to http://fedora-ingest-rails-qa-19310744.us-east-1.elb.amazonaws.com/changelog (the migrated environment) and sign into the changelog. You'll see the current item listed.
